open-company/open-company-web
# Truncate notifications

Card: https://trello.com/c/sYFRdHaY

To test:
- add a post with user A
- with user B add a very long comment
- [x] user A sees only 3 lines of text in the bottom left corner? Good
- now with user A open the notifications tray
- [x] do you see the new notification truncated?
- switch to mobile
- [x] repeat (no notifications tray on mobile)
